references/cli/status.go

```diff
@@ -118,6 +136,9 @@ func NewAppStatusCommand(c common.Args, order string, ioStreams cmdutil.IOStream
 			if err != nil {
 				return err
 			}
+			if outputFormat != "" {
+				return printRawApplication(context.Background(), c, outputFormat, cmd.OutOrStdout(), namespace, appName)
```

Comment:
Does "showEndpoints" still work? if "return" here.

Reply:
No, this is intended. 

If the user wants raw output, then the endpoints table will be purely noise to them.